dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceService.java

```diff
@@ -1567,7 +1576,10 @@ public synchronized void println(char c[]) {
         public synchronized void println(String s) {
             TrOutputStream.isPrinting.set(true);
             try {
-                super.print(s);
+                if (TrOutputStream.isPrintingStackTrace.get())
+                    super.println(s);
```

Comment:
I believe our code in filterStackTraces is stripping the NLs off of the lines in the stack trace, which is necessitating this unusual need for println.  would be better if we could avoid stripping the NLs in the first place so that this isn't needed.

Reply:
`filterStackTraces` already seems to be donig it.
Changing it back to `print()` doesn't seem to take any effect.